src/sentry/utils/pytest/selenium.py

```diff
@@ -332,6 +333,10 @@ def browser(request, percy, live_server):
     driver.set_window_size(window_width, window_height)
 
     def fin():
+        # dump console log to stdout, will be shown when test fails
+        for entry in driver.get_log("browser"):
+            sys.stderr.write(repr(entry))
```

Comment:
Does this include enough context for folks to know where it came from?

Reply:
I'm not sure how to answer the question, I guess that depends on the issue you're debugging. It helped me a bunch of times for sure.

Reply:
Could we include a prefix or line above indicating that this is the console.log content?